DCCpp_Uno/src/CurrentMonitor.h

```diff
@@ -12,7 +12,7 @@ Part of DCC++ BASE STATION for the Arduino
 
 #include "Arduino.h"
 
-enum MOTOR_BOARD_TYPE { ARDUINO_SHIELD, POLOLU, BTS7960B_5A, BTS7960B_10A };
+enum MOTOR_BOARD_TYPE { ARDUINO_SHIELD, POLOLU, BTS7960B_5A, BTS7960B_10A, LMD18200, LMD18200_MAX471 };
```

Comment:
Should we move Motor board type to a separate class which has all the variables associated with each board so we can readily pull them once a board is selected via a keyvalue pair